### ISSUE TITLE
Errors running `npm install`

### ISSUE THREAD

**Issue by C4Framework** [SwiftDocOrg/swiftdoc-parser#5]
I get the following errors at the end of running `nom install`

```
npm ERR! Darwin 14.3.0
npm ERR! argv "node" "/usr/local/bin/npm" "install"
npm ERR! node v0.12.2
npm ERR! npm  v2.7.4
npm ERR! code ELIFECYCLE

npm ERR! pcre@0.0.6 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the pcre@0.0.6 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the pcre package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls pcre
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/travis/Development/swiftdoc-parser/npm-debug.log
```

There's a ton of other messages that look iffy as well. Here's a link to my terminal output:
https://www.dropbox.com/s/c3uifbsabdsrdsk/terminalOutput.txt?dl=0

Here's a link to the npm log:
https://www.dropbox.com/s/5yog82jpyn3mc9h/npm-debug.log?dl=0

**Comment by natecook1000** [comment]
From what I can tell that's an issue with `node-gyp`—can you try the remedy listed on this page and see if that solves it for you?

https://github.com/TooTallNate/node-gyp/wiki/Updating-npm%27s-bundled-node-gyp

**Comment by C4Framework** [comment]
hey, sorry for the delay, went on vacation and just got back.

I tried the update to `node-gyp`, that resulted in the same error after running `npm install` again.

Then, I updated versions of node, npm, etc.

```
node-gyp@2.0.2
node@0.12.7
pcre@8.37
dot - graphviz version 2.38.0 (20140413.2041)
```

**Comment by tbondwilkinson** [comment]
I got the same errors, and fixed them by using node v0.10 rather than 0.12.  v0.11 would likely also work.

**Comment by huy-le** [comment]
I got this issue again. I download a stable nodejs from https://nodejs.org/en/ then updated npm by `sudo npm install npm -g`.

But got a bunch of errors when run npm install.
